src/components/layout/Header.tsx

```diff
@@ -7,7 +7,7 @@ const Wrapper = styled.header`
   color: #ffffff;
   background-color: #3340a9;
   min-height: ${rem('325px')};
-  font-size: 1.5am;
+  font-size: 1.5em;
```

Comment:
This is messing with the size... 
Compare:
* https://pr82--dexreact.review.gnosisdev.com/
* http://dex.dev.gnosisdev.com

I think we can just remove the line.

Reply:
removed